code/main/edge/locationHintManager.brs

```diff
@@ -15,12 +15,23 @@
 
 function _adb_LocationHintManager() as object
 
-    return {
+    locationHintManager = {
         _DEFAULT_LOCATION_HINT_TTL_SEC: 1800,
         _EDGE_NETWORK_SCOPE: "edgenetwork",
 
         _locationHint: invalid,
-        _timer: invalid,
+        _initTSInMillis&: _adb_InternalConstants().TIMESTAMP.INVALID_VALUE,
```

Comment:
Sorry for missing this earlier. I don’t believe we need to store or perform any additional checks based on initTS. Could you please remove it?